index.bs

```diff
@@ -247,7 +247,13 @@ When asked to <dfn abstract-op>remove client hints from redirect if needed</dfn>
   <a href="#policy-controlled-features">associated
   policy-controlled feature</a>, returns <code>false</code>, then remove <var>hintName</var> from
   <a for=request>header list</a>.
-  [[!PERMISSIONS-POLICY]]
+  <li><p>Else if <var>request</var>'s <a for=request>header list</a> <a for="header list">doesn't contain</a>
+  <var>hintName</var> and if the result of running <a
+  href="https://w3c.github.io/webappsec-permissions-policy/#algo-should-request-be-allowed-to-use-feature">Should
```

Comment:
This is [exported](https://github.com/w3c/webappsec-permissions-policy/blob/main/index.bs#L1008), so you should be able to [= foo bar =] it, rather than hot link, or do what we do in https://github.com/WICG/client-hints-infrastructure/blob/main/index.bs#L224).

(I know this was there, but can you clean it up while you're poking at this?)